apiserver/facades/controller/charmrevisionupdater/interface.go

```diff
@@ -37,13 +38,18 @@ type Application interface {
 type Model interface {
 	CloudName() string
```

Comment:
CloudName, CloudRegion, ModelTag and UUID can all be removed as well if you are after a drive by?

If we call the model info service inside the register call and get the model we are working on as a coremodel.ReadOnlyModel we can hold that struct type on the api and pull this information off of that for the life of the facade.

This doesn't need to be done for model config epic but it is part of the model epic that is up next.

Reply:
I'll try to remember for the next work. It's a bit big for a drive by.

Reply:
I'm going to leave the work for the model epic. We need to get model-config done.